### DISCUSSION TITLE
Collaboration Design Patterns > Affecting others > Piggy-Backing

### DISCUSSION THREAD

**Discussion by giscus** [pgpais/live-framework-for-cooperative-games#107] [Framework Discussions]
# Collaboration Design Patterns > Affecting others > Piggy-Backing

https://www.lfcooperativegames.com/reports/new

**Comment by hg65we** [comment]
Since it's a team-based game mode, a player is able to not do a single thing in the game and still win it as long as the team gains the necessary points or destroys all enemy ships
